projects/arlas-toolkit/src/lib/components/change-password/change-password.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="change-title" mat-dialog-title>
+  {{'Change password' | translate}}
+</div>
+<div mat-dialog-content class="change-container">
+  <div class="change-form text-center" style="align-items: center;">
```

Comment:
can the style be included in one of the classes ?

Reply:
Fixed